services/blockstorage/service.go

```diff
@@ -82,7 +162,15 @@ func (s *service) GetLastCommittedBlockHeight(input *services.GetLastCommittedBl
 }
 
 func (s *service) ValidateBlockForCommit(input *services.ValidateBlockForCommitInput) (*services.ValidateBlockForCommitOutput, error) {
-	panic("Not implemented")
+	if protocolVersionError := s.validateProtocolVersion(input.BlockPair); protocolVersionError != nil {
+		return &services.ValidateBlockForCommitOutput{}, protocolVersionError
```

Comment:
why not return nil on error (we are working with pointers here, it should work)

Reply:
I prefer not to have null pointer dereference